src/components/MotionDiv.tsx

```diff
@@ -0,0 +1,15 @@
+import { motion } from 'framer-motion';
+import type { ReactNode } from 'react';
+
+export const MotionDiv = ({ children }: { children: ReactNode }) => {
```

Comment:
I noticed we still use `motion.div` in other parts of the code like `Hero.ts` and `PromoVideo.ts` with different configurations and if refactor this MotionDiv to accept props then we'll just end up creating an unnecessary wrapper around `motion.div` so I think it's best to leave it as is. Thanks so much for the contribution, really appreciate it!

Reply:
Understood good luck ✅